src/lib/components/drs/pages/DrsObjectForm.js

```diff
@@ -100,14 +105,33 @@ const DrsObjectForm = (props) => {
 
             <Grid container justify='space-between' alignItems='center'>
                 <Grid item xs={2} align='left'>
-                    <Button variant='contained' component={Link} to='/drs' color='primary' size='large'>
-                        <Typography variant='button'>DRS Index</Typography>
-                    </Button>
+                    {
+                        props.formViewType === FormViewType.EDIT 
+                        ?   <DeleteDrsObjectButton {...p.delete} setError={setError}/>
+                        :   <Button variant='contained' color='primary' size='large'
+                            component={Link} to='/drs' onClick={scrollToTop}>
+                                <Typography variant='button'>DRS Index</Typography>
+                            </Button>
+                    }
                 </Grid>
                 <Grid item xs={8}>
                     <Typography align='center' variant="h3" gutterBottom>{props.title}</Typography>
                 </Grid>
-                <Grid item xs={2} />
+                <Grid item xs={2} align='right'>
+                    {
+                        props.formViewType === FormViewType.SHOW 
```

Comment:
avoid nested ternary statements if possible, since they get a bit visually confusing IMO. Replace with 2 unlinked ternary statements, one for SHOW that renders nothing if not SHOW, and one for EDIT that renders nothing if not EDIT

i.e.
```
SHOW ? showComponent : null
EDIT ? editComponent : null
```